src/Nethermind/Nethermind.Evm/TransactionProcessing/BuildUpTransactionProcessorAdapter.cs

```diff
@@ -31,5 +31,8 @@ public BuildUpTransactionProcessorAdapter(ITransactionProcessor transactionProce
         
         public void Execute(Transaction transaction, BlockHeader block, ITxTracer txTracer) =>
             _transactionProcessor.BuildUp(transaction, block, txTracer);
+        
```

Comment:
missing the point, adapters where introduced to segregate the type of transaction execution

Reply:
Removed unnecessary method.